back-end/init.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(50) UNIQUE NOT NULL,
+  email VARCHAR(50) UNIQUE NOT NULL,
+  first_name VARCHAR(50),
+  last_name VARCHAR(50)
+);
+
+CREATE TABLE sounds (
+  id SERIAL PRIMARY KEY,
+  user_id INTEGER NOT NULL REFERENCES users(id),
+  title VARCHAR(50),
+  path VARCHAR(200) UNIQUE NOT NULL
```

Comment:
Not sure if it matters or not, but maybe we can name this filepath instead of path?